docs/install/index.rst

```diff
@@ -5,6 +5,6 @@ RestrictedPython is usually not used stand alone, if you use it in context of yo
 
 For a standalone usage:
 
-.. code:: bash
+.. code-block:: shell
```

Comment:
please use console instead of shell as this is a console session block see: http://pygments.org/docs/lexers/#lexers-for-various-shells